kotlin/query-with-cql-filters/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Search and Query",
+    "description": "Query data from an OGC API feature service using CQL filters.",
+    "formal_name": "QueryWithCqlFilters",
+    "ignore": false,
+    "images": [
+        "query-with-cql-filters.png"
+    ],
+    "keywords": [
+        "CQL",
+        "OGC",
+        "OGC API",
+        "browse",
+        "catalog",
+        "common query language",
+        "feature table",
+        "filter",
+        "query",
+        "service",
+        "web",
+        "OgcFeatureCollectionTable",
+        "QueryParameters",
+        "TimeExtent"
+    ],
+    "language": "kotlin",
+    "relevant_apis": [
+        "OgcFeatureCollectionTable",
+        "QueryParameters",
+        "TimeExtent"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgisruntime/sample/query_with_cql_filters/MainActivity.kt"
```

```suggestion
        "src/main/java/com/esri/arcgisruntime/sample/querywithcqlfilters/MainActivity.kt"
```

Can we check the folder path too to make sure it hasn't got dashes